OpenUtau/Styles/Styles.axaml

```diff
@@ -62,7 +62,7 @@
     <Setter Property="Padding" Value="4,1"/>
     <Setter Property="Margin" Value="0,0,0,4"/>
     <Setter Property="FontSize" Value="12"/>
-    <Setter Property="FontFamily" Value="{DynamicResource ui.textbox.fontfamily}"/>
+    <Setter Property="FontFamily" Value="{DynamicResource ui.fontfamily}"/>
```

Comment:
These was for working around Avalonia 10 could not handle CJK font fallback in text boxes. You can simply delete this line and delete `ui.textbox.fontfamily` from `Strings.axaml`

Reply:
Ok! updated.